mongo_connector/doc_managers/neo4j_doc_manager.py

```diff
@@ -53,7 +52,26 @@ def upsert(self, doc, namespace, timestamp):
         "ns": namespace,
         "_ts": timestamp
     }
+    doc = self._formatter.format_document(doc)
+    self.store_nodes_and_relationships(doc, doc_type, doc_id)
+    
   
+  def store_nodes_and_relationships(self, doc, doc_type, doc_id):
+    tx = self.graph.cypher.begin()
+    parameters = {'id':doc_id}
+    for key in doc.keys():
+      LOG.error(type(doc[key]))
+      if (type(doc[key]) is str):
+        value = doc[key]
+      else:
+        value = ""
```

Comment:
As I mentioned here, https://github.com/hannelita/neo4j_doc_manager/commit/6214d585375b0b6d63c4e030d1296c6a33a1ee48#commitcomment-12627294 , This will be changed for relationship handling